packages/react-components/react-tree/stories/Tree/TreeWithoutInlineStyle.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  Tree,
+  TreeItem as BaseTreeItem,
+  TreeItemLayout,
+  treeItemLevelToken,
+  TreeItemProps,
+  useTreeContext_unstable,
+} from '@fluentui/react-tree';
+import { makeStyles, mergeClasses } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  level1: {
+    [treeItemLevelToken]: 1,
+  },
+  level2: {
+    [treeItemLevelToken]: 2,
+  },
+  level3: {
+    [treeItemLevelToken]: 3,
+  },
+});
+
+const TreeItem = (props: TreeItemProps) => {
+  const styles = useStyles();
+  const level = useTreeContext_unstable(ctx => ctx.level);
+  return (
+    <BaseTreeItem
+      {...props}
+      style={{ [treeItemLevelToken]: undefined, ...props.style }}
```

Comment:
```suggestion
      style={{ [treeItemLevelToken]: undefined }}
```

we don't need to forward `style` here right? the audience for the docs is app developers